actions.py

```diff
@@ -42,6 +42,16 @@ def sizeof_fmt(num, suffix="B"):
 		num /= 1024.0
 	return "%.1f%s%s" % (num, "Yi", suffix)
 
+def album_info(release):
```

Comment:
it is great practice to have a printing function, but I would maybe try to make it more abstract and general. I believe this could be easily encoded in a constant dictionary, for example, and the function would just iterate over them. It's also easier to maintain.

Reply:
also, the function name could be a little more clear. Maybe `print_album_info()`?